pkg/util/globalerror/errors.go

```diff
@@ -101,6 +102,11 @@ func (id ID) LabelValue() string {
 	return strings.ReplaceAll(string(id), "-", "_")
 }
 
+// Error implements error.
+func (id ID) Error() string {
```

Comment:
Note to reviewers: made it implement `error` interface so that I can use it in the ingester's `handleAppendError()`.